### PR TITLE
upgrade v2022-08

### PR DESCRIPTION
**Pull request by gwbres**
Recommend a newer BR tag, `volk` is officially integrated, resolve conflict

### CODE REVIEW

**Comment by gwbres** [comment]
Salut Gwen @trabucayre 
sur quelles archi tu utilise les customisations noyaux linux/  ?
elles ne sont activées par aucune config ou aucun noyau_defconfig

a+

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ the images to be found at the end of the compilation in <code>output/images</cod
 
 ### Scipy
 
-`package/python-scipy` is now officially integrated. Use latest Buildroot for the following.
+`package/python-scipy` is now officially integrated.
```
Comment (trabucayre) [inline README.md]: Why not removing the entire line?

**Comment by trabucayre** [comment]
LGTM.
A tag with current buildroot version must be added to freeze repo's status before the bump.

File: README.md (hunk 1 above)
Comment (gwbres) [reply README.md]: > Why not removing the entire line?

Yes, it makes more sense, we are "configurations" provider from now on, no need to mention the package in itself

**Comment by trabucayre** [comment]
Applied

**Pull request merged**